src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -579,4 +560,30 @@ class JavaMigrations {
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.6-open", _))
   }
 
+	@ChangeSet(order = "182", id = "182-add_graalvm_19_2_0", author = "eddumelendez")
+	def migrate182(implicit db: MongoDatabase) = {
+		List(
+			Version(
+				candidate = "java",
+				version = "19.2.0-grl",
+				url = "https://github.com/oracle/graal/releases/download/vm-19.2.0/graalvm-ce-linux-amd64-19.2.0.tar.gz",
+				platform = Linux64,
+				vendor = Some(Graal)),
+			Version(
+				candidate = "java",
+				version = "19.2.0-grl",
+				url = "https://github.com/oracle/graal/releases/download/vm-19.2.0/graalvm-ce-darwin-amd64-19.2.0.tar.gz",
+				platform = MacOSX,
+				vendor = Some(Graal)),
+			Version(
+				candidate = "java",
+				version = "19.2.0-grl",
+				url = "https://github.com/oracle/graal/releases/download/vm-19.2.0/graalvm-ce-windows-amd64-19.2.0.zip",
+				platform = Windows,
+				vendor = Some(Graal)))
+			.validate()
+			.insert()
+		Seq(Linux64, MacOSX).foreach(platform => removeVersion(candidate = "java", version = "19.1.1-grl", platform))
```

Comment:
Please do not remove old versions, specially if they are from another minor version. Some tools and frameworks do not support the latest version and still need to use previous one.

Reply:
@ilopmar thanks for letting me know about it. that version is not going to be removed then